scripts/release/prod/stage/snapshot.sh

```diff
@@ -1,13 +1,35 @@
 #!/usr/bin/env bash
 
```

Comment:
Add file header.

Reply:
This isn't a cli tool.  Also, it's clear from the variable definitions and the conditional what the expected inputs are.